persistence/upgrades/ravendb-3to4.md

```diff
@@ -37,6 +37,21 @@ NServiceBus now uses the asynchronous RavenDB API for all operations. If sharing
 Previously the API exposed an [`IDocumentSession`](https://ravendb.net/docs/search/latest/csharp?searchTerm=IDocumentSession), but now exposes [`IAsyncDocumentSession`](https://ravendb.net/docs/search/latest/csharp?searchTerm=IAsyncDocumentSession) instead, which contains the same operations but using a Task-based API.
 
 
+## Migrate saga data if using AllowStaleSagaReads
+
+Due to changes in how NServiceBus Version 6 handles saga correlation properties, solutions that previously used the `AllowStaleSagaReads()` option in RavenDB Persistence will not work properly and need to be migrated during upgrade.
```

Comment:
"changes in how NServiceBus Version 6 handles saga correlation properties" assuming is should be possible to add a more info link here?